### DISCUSSION TITLE
Windows 10 Portable install. Running but not fully.

### DISCUSSION THREAD

**Discussion by AlanEdd** [espanso/espanso#2032] [Q&A]
Hey there,
I installed as instructed in a portable location. Sets up fine but the triggers are not being detected. eg. typing :date will not change in another app. The pop up works fine and the icon is there in the tray and functional. It is not disabled etc. In the log there is one Warning:

`[worker(34684)] [WARN] unable to find espanso AppUserModelID in the list of registered ones, falling back to Powershell`

Any pointers on what to try next ?
Thanks a lot.
A

**Comment by smeech** [comment]
Linked with #1818

**Reply by AlanEdd** [reply]
Thank you.
So I have been playing with the config. If I insert (so I can see what is happening) and not sure what the second flag does really, I was just iterating through settings until I saw something different.

`key_delay: 500`
`win32_exclude_orphan_events: false`

Now  I type ` :date`
So it is replacing it with the date (less last digit) and then replacing it back again with :date. 
I can see that it is replaced   `:date`    -> `31/07/202` -> ` 3:date`


So that might help a bit.
Thanks.

**Reply by AlanEdd** [reply]
And now it's working but I have no idea why or what I changed.
 Here is my config for reference as I was experimenting with flags.
```

enable: true
apply_patch: true

#Toggle key on/off------------------------------------------------------------------
#espanso (when double tapped)
# Available options: CTRL, SHIFT, ALT, CMD, OFF, META (windows key.)
#and LEFT_CTRL, RIGHT_SHIFT, etc...
toggle_key: ALT
# toggle_key: OFF

#SHORTCUT override-----------------------------------------------------------------
search_shortcut: ALT+SPACE
# search_shortcut: off

# Search Trigger-------------------------------------------------------------------
# Pop up quicker when typing ! same as hotkey really.
search_trigger: "jk"
# search_trigger: none


# Show tray icon.
show_icon: true

keyboard_layout: { layout: us }

show_notifications: true

# win32_exclude_orphan_events: false

# --- Injection Backend

# Espanso supports multiple ways of injecting text into applications. Each of
# them has its quirks, therefore you may want to change it if you are having problems.
# By default, espanso uses the "Auto" backend which should work well in most cases,
# but you may want to try the "Clipboard" or "Inject" backend in case of issues.
backend: auto

# Enable/disable the config auto-reload after a file change is detected.
auto_restart: true

evdev_modifier_delay: 5
key_delay: 5
```

**Comment by AlanEdd** [comment]
Ok I think  I have something,  I have been using `CopyQ `software and once I kill that process the hit and miss nature of Espanso seems to be gone and all expansions are working.

https://hluk.github.io/CopyQ/

Thanks 
A
